narratives/ncov_sit-rep_2020-04-10.md

```diff
@@ -0,0 +1,6369 @@
+
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Expansion of North American and European outbreaks](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&p=full&d=tree,map)
+
+During the 3rd month of the outbreak, from mid-February to mid-March 2020, these two clusters expanded to cause large outbreaks within Europe and North America, respectively.
+<!-- There is no right side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Spread across international borders](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&f_region=North%20America&p=full&r=division&d=tree,map&c=division)
+
+Within each of these regions, we see widespread mixing across state lines and international borders.
+This is demonstrated here by the intermingling of samples from multiple U.S. states and Canadian provinces.
+<br><br>
+We've [previously reported](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?f_region=Europe&n=5) on a similar pattern of frequent mixing of viral lineages between European countries.
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Introductions from North America & Europe to other parts of the globe](https://nextstrain.org/ncov/global/2020-04-09?dmax=2020-03-15&dmin=2020-02-15&f_region=Africa,South%20America,Oceania&p=full&d=tree,map)
+
+These clusters are not restricted to their respective regions. During this time period, we also see cases from South America, Oceania, and Africa against a backdrop of primarily European and North American clades. This indicates that there were frequent intercontinental introductions during this time period.
+<br><br>
+Interestingly, samples from the global south span the full breadth of the tree, indicating that these regions' outbreaks are mixtures of many viral lineages.
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Coming full circle: reintroductions to Asia](https://nextstrain.org/ncov/global/2020-04-09?d=tree,map&dmin=2020-03-14&f_region=Asia&p=full&r=division)
+
+In the last month, the pandemic has come full circle. Here, we see re-introductions from Europe and North America back to Asia.
+<br><br>
+This resoundingly demonstrates that this pandemic is a global fight; we all must help control this virus everywhere if we hope to defeat it anywhere.
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 13-->
+# [Takeaways](https://nextstrain.org/ncov/2020-03-27?c=country&d=map&p=full)
+<br>
+<!-- This is the right-side text -->
+
+```auspiceMainDisplayMarkdown
+# [What you can do](https://nextstrain.org/ncov/2020-03-27?c=country&d=map&p=full)
```

Comment:
```suggestion
# What you can do
```
This is causing the page to not show correctly - technically this is a header for a new 'slide'! So I'm having this split in half in a super-weird way :)